src/realm_dart_sync.h

```diff
@@ -35,3 +38,9 @@ RLM_API void realm_dart_sync_connection_state_changed_callback(realm_userdata_t
                                                                realm_sync_connection_state_e new_state);
 
 RLM_API void realm_dart_sync_on_subscription_state_changed_callback(realm_userdata_t userdata, realm_flx_sync_subscription_set_state_e state);
+
+RLM_API bool realm_dart_sync_before_reset_handler_callback(realm_userdata_t userdata, realm_t* realm);
+
+RLM_API bool realm_dart_sync_after_reset_handler_callback(realm_userdata_t userdata, realm_t* before_realm, realm_thread_safe_reference_t* after_realm, bool did_recover);
+
+RLM_API void realm_dart_invoke_navite_with_result(bool success, void* unlockFunc);
```

Comment:
```suggestion
RLM_API void realm_dart_invoke_native_with_result(bool success, void* unlockFunc);
```
Need to run ffigen again